codespell_lib/data/dictionary.txt

```diff
@@ -2394,6 +2397,7 @@ buss->bus
 busses->buses
 bussiness->business
 butons->buttons
+buts->but
```

Comment:
Suggest butts

Reply:
Valid in the US? https://dictionary.cambridge.org/dictionary/english/buts

Reply:
Also "no ifs no buts"

Reply:
Done

Reply:
Sorry, I think after my rambling and given the Cambridge hit this should probably be removed @Gelma .

Reply:
Removed